Source/ACE.Entity/Enum/WeenieType.cs

```diff
@@ -75,6 +75,7 @@ public enum WeenieType : uint
         SocialManager,
         Pet,
         PetDevice,
-        CombatPet
+        CombatPet,
+        Foci
```

Comment:
We align the formatting on all of the other files - you want it unaligned?   I am not sure I follow?